Sources/Core/Transaction/Policies.swift

```diff
@@ -53,13 +53,5 @@ public struct Policies {
             self.maxPriorityFeePerGasPolicy = maxPriorityFeePerGasPolicy
         }
 
-    public static var auto: Policies {
-        return Policies(
-            noncePolicy: .latest,
-            gasLimitPolicy: .automatic,
-            gasPricePolicy: .automatic,
-            maxFeePerGasPolicy: .automatic,
-            maxPriorityFeePerGasPolicy: .automatic
-        )
-    }
+    public private(set) static var auto = Policies()
```

Comment:
Can be declared as `public static let auto = Policies()` since it's no longer a computed variable.